common/transforms/Cyrillic-Latin.xml

```diff
@@ -33,7 +33,12 @@ $descender = ˌ;
 $umlaut = \u0308 ;
 $macron = \u0304 ;
 $breveBelow = \u032E;
```

Comment:
Is this unused now?

Reply:
Just checked, and it isn't used. (I don't think this is worth a new PR; we haven't been concerned about details like that in the files.